metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchQueryBuilder.java

```diff
@@ -32,6 +32,9 @@
 
 
 public class SearchQueryBuilder {
+
+  public static final String STRUCTURED_QUERY_PREFIX = "\\\\/q ";
+  public static final float EXACT_MATCH_BOOST_FACTOR = 10.0f;
```

Comment:
Exact match on ANY field?

Reply:
It's a multiplier, so the annotation * this factor.